Update availability-zones.md (#100989)
Azure/azure-content: articles/aks/availability-zones.md

When you create a cluster using the [az aks create][az-aks-create] command, the `--zones` parameter defines which zones agent nodes are deployed into. The control plane components such as etcd or the API are spread across the available zones in the region if you define the `--zones` parameter at cluster creation time. The specific zones which the control plane components are spread across are independent of what explicit zones are selected for the initial node pool.

> If you don't define any zones for the default agent pool when you create an AKS cluster, control plane will not be in Availability Zones.

Reviewer: Suggest rewording to - If you don't define any zones for the default agent pool when you create an ASK cluster, the control plane components won't be present in availability zones.

Author: It is perfect for me. I'm not an English native speaker, feel free to change the text!! :)

> If you don't define any zones for the default agent pool when you create an AKS cluster, control plane will not be in Availability Zones.

Reviewer 2: ```suggestion
If you don't define any zones for the default agent pool when you create an ASK cluster, the control plane components won't be present in availability zones.
```
Adding Matt's suggestion, per approval from contributor

> If later, you add additional node pools using the az aks nodepool add command and specify --zones for new nodes, the Control Plane will be converted to Availability Zones.

Reviewer: Suggest rewording to - Later if you add additional node pools using the `az aks nodepool add` command with the `--zones` argument for new nodes, the AKS control plan is converted to spread across availability zones.

Author: It is perfect for me. I'm not an English native speaker, feel free to change the text!! :)

> If later, you add additional node pools using the az aks nodepool add command and specify --zones for new nodes, the Control Plane will be converted to Availability Zones.

Reviewer 2: ```suggestion
Later if you add additional node pools using the `az aks nodepool add` command with the `--zones` argument for new nodes, the AKS control plan is converted to spread across availability zones.
```
Adding Matt's suggestion, per approval from contributor
